### PR TITLE
Export ignore

### PR DESCRIPTION
**Pull request by lippserd**
The main topic of this PR is removing scripts, RELEASE.md and Git-related files from release tarballs.
It also includes minor changes to the make-release script.

### CODE REVIEW

File: bin/make-release.sh [hunk 1]
```diff
@@ -43,10 +43,16 @@ find vendor/ -type f -name "*.php" \
  | xargs -L1 git add -f
 find vendor/ -type f -name LICENSE | xargs -L1 git add -f
 find vendor/ -type f -name '*.json' | xargs -L1 git add -f
-find asset/ -type f | xargs -L1 git add -f
+find -L asset/ -type f | xargs -L1 git add -f
```
Comment (nilmerg) [inline bin/make-release.sh]: Why is this necessary? The assets are copied over, not symlinked.

File: bin/make-release.sh [hunk 2]
```diff
@@ -43,10 +43,16 @@ find vendor/ -type f -name "*.php" \
...
 echo "v$VERSION" > VERSION
 git add VERSION
 git add composer.lock -f
+cat <<gitattributes >>.gitattributes
+.git* export-ignore
+bin/ export-ignore
+RELEASE.md export-ignore
+gitattributes
+git add .gitattributes
```
Comment (nilmerg) [inline bin/make-release.sh]: Why is this not a file in git?

Comment (lippserd) [reply bin/make-release.sh]: I thought it was okay to still have them when archives are being created from the master branch.

File: bin/make-release.sh (hunk 1 above)
Comment (lippserd) [reply bin/make-release.sh]: You're right! Maybe I was using an older version with this change?

File: bin/make-release.sh (hunk 2 above)
Comment (nilmerg) [reply bin/make-release.sh]: Why should anyone do so? The readme states that the master branch is not usable. And if one wants to create a new release, publishing it requires git access anyway.

**Comment by lippserd** [comment]
Closing here. I added relevant commit from this PR to #7.
